2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1022,3 +1022,82 @@ Questa attività consiste nei seguenti compiti:
 - implementare e monitorare misure per valutare l'efficacia dei trattamenti dei rischi;
 
 - monitorare continuamente l'emergere di nuovi rischi e fonti durante l'intero ciclo di vita.
+
+
+== Processo di Controllo della Qualità
+
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.8_
+
+=== Scopo
+
+La International Software Testing Qualifications Board (ISTQB) definisce il controllo della qualità come "attività incentrate sul garantire che i requisiti di qualità saranno soddisfatti".
+
+Il processo di controllo della qualità ha quindi come obiettivo principale quello di garantire l'applicazione efficace del processo di gestione della qualità da parte del gruppo. 
+
+Il suo focus fondamentale è fornire la sicurezza e la certezza che i requisiti di qualità stabiliti saranno pienamente soddisfatti. 
+Questo viene ottenuto mediante un'analisi proattiva attenta dei processi e dei risultati che caratterizzano il ciclo di vita del progetto. 
+
+L'obiettivo è assicurare che il prodotto risponda alle aspettative in termini di qualità, e che ciò avvenga in piena conformità con le linee guida, le politiche e le procedure precedentemente definite. 
+In altre parole, la finalità ultima è garantire che la qualità desiderata sia intrinseca al prodotto finale e che ogni fase del processo segua gli standard e le disposizioni stabilite dal gruppo e dal progetto.
+
+=== Risultati
+
+L'implementazione riuscita del processo di controllo della qualità produce i seguenti risultati:
+
+- le procedure di controllo della qualità del progetto sono definite e implementate;
+- vengono definiti criteri e metodi per la valutazione di controllo della qualità;
+- vengono eseguite valutazioni dei prodotti, servizi e processi del progetto;
+- i risultati delle valutazioni vengono forniti agli stakeholders;
+- i problemi vengono trattati.
+
+=== Attività
+
+Le seguenti attività devono essere implementate in conformità con le politiche e le procedure del gruppo:
+
++ prepararsi per il controllo della qualità:
```

Comment:
c'è un problema di comprensibilità piuttosto grave in tutti i punti di questo elenco puntato.
se compili e guardi noterai che non si capisce quale lista e sottolista di cosa. Il problema secondo me si può risolvere cambiando i marker delle liste di livello superiore rispetto a quelle di livello inferiore, andando a definire tanti marker quanti sono i livelli di annidamento delle liste. 
Potrebbe essere utile il paragrafo "marker" di questa pagina
https://typst.app/docs/reference/model/list/

p.s. attento anche all'indentazione delle liste più interne

Reply:
![immagine_2023-12-30_193343572](https://github.com/Error-418-SWE/Documenti/assets/91839513/5b0554be-1932-4adf-abc8-d378e8393282)

secondo me arrivati al terzo livello di annidamento della lista puntata usando sempre lo stesso marker non si capisce bene, sembra che sia impazzita l'indentazione. Ho provato ad utilizzare il + al posto del - ed esce un marker del tipo 
a)
b)
...

e secondo me non è male

Reply:
<img width="306" alt="Immagine 2023-12-30 194001" src="https://github.com/Error-418-SWE/Documenti/assets/91839513/79da45c4-25be-45b0-885d-0ce2ce062304">